apps/cpp_rpc/rpc_env.cc

```diff
@@ -153,13 +153,13 @@ RPCEnv::RPCEnv(const std::string& wd) {
 }
 /*!
  * \brief GetPath To get the work path from packed function
- * \param file_name The file name
+ * \param file_path The file path
  * \return The full path of file.
  */
-std::string RPCEnv::GetPath(const std::string& file_name) const {
```

Comment:
If we decide to change the name of function argument, we should change header `rpc_env.h` and relative function document too. However, should we change `file_name` into `file_path`? As the purpose of this function is to get the right path of one specific file name (no matter it is in the relative path or exact full path). It is a bit strange we pass `file_path` and return `file_path`.

Reply:
Changed `file_path` back to `file_name`